Tasks/AzureCloudPowerShellDeploymentV1/task.json

```diff
@@ -27,6 +27,11 @@
             "name": "newServiceAdvancedOptions",
             "displayName": "Advanced Options For Creating New Service",
             "isExpanded": false
+        },
+        {
+            "name": "advancedSettings",
```

Comment:
Why can't we use the same group above, which says "Advanced options for creating new service"? We should avoid multiple "Advanced" groups...

Reply:
Because It's related to the deployment of service instead of just creation of service and all arguments belonging to the group  "Advanced options for creating new service" are related to the creation of cloud service.